core/include/moveit/task_constructor/solvers/cartesian_path.h

```diff
@@ -51,6 +52,7 @@ class CartesianPath : public PlannerInterface
 {
 public:
 	CartesianPath();
+	CartesianPath(const kinematics::KinematicsQueryOptions& options);
```

Comment:
Could you please make a property similar to `step_size`, `jump_threshold`, and `min_fraction`?

Reply:
done